### ISSUE TITLE
RatScanner crashing when name scanning

### ISSUE THREAD

**Issue by terrasearch** [RatScanner/RatScanner#104]
**Describe the bug**
RatScanner crashes when I press on the magnifying glass for name scanning

**To Reproduce**
1. Press magnifying glass on any item
2. Rat Scanner crashes

**Expected behavior**
Showing item and value

**Additional information**
- Windows 10 x64
- 3 Monitors
- Resolution of game 1920x1080

**Log file content**
```
[02:35:12.0249367] > [Debug] OnKeyUp:96 -> Pressed: Left
[02:35:12.0252030] > [Debug] NameScan:218 -> Name scanning at: (1220, 389)
[02:35:13.5047452] > [Debug] OnKeyUp:96 -> Pressed: Left
[02:35:13.5050459] > [Debug] NameScan:218 -> Name scanning at: (1312, 122)
[02:35:13.6677615] > [Debug] OnKeyUp:96 -> Pressed: Left
[02:35:13.6679478] > [Debug] NameScan:218 -> Name scanning at: (1312, 122)
[02:35:14.8847377] > [Debug] OnKeyUp:96 -> Pressed: Left
[02:35:14.8848939] > [Debug] NameScan:218 -> Name scanning at: (634, 299)
[02:35:14.9653352] > [Debug] .ctor:71 -> Marker found! IsShortName: False
[02:35:14.9727415] > [Debug] OCR:162 -> Scaling image...
[02:35:14.9803407] > [Debug] OCR:184 -> Gray scaling...
[02:35:14.9804577] > [Debug] OCR:188 -> Binarizing...
[02:35:14.9805947] > [Debug] OCR:192 -> Applying OCR...
```

**Comment by Blightbuster** [comment]
Hey, are you using version 2.7.0?

**Comment by terrasearch** [comment]
Yes, I'm using 2.7.0

**Comment by Blightbuster** [comment]
Hey, please try with version [v2.8.0](https://github.com/RatScanner/RatScanner/releases/tag/v2.8.0) as it uses a new implementation of name scanning.

**Comment by Blightbuster** [comment]
Fixed with v2.8.5
